### PR TITLE
feature: Add all pages as uncategorized

### PR DESCRIPTION
**Pull request by mrfyda**
This should be a first step in moving all articles to the new platform.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -12,7 +12,6 @@ theme:
     text: 'Ubuntu'
     code: 'Ubuntu Mono'
   logo: 'images/codacy-logo.png'
-  custom_dir: 'theme'
```
Comment (pauloribeiro-codacy) [inline mkdocs.yml]: We will need this line when we start customizing the theme for the documentation (all the HTML and CSS that overrides the default theme need to be placed in this directory). But it's OK if we remove it for now.

**Comment by pauloribeiro-codacy** [comment]
Hi @mrfyda!

Thanks for this contribution! :muscle: 

What is the process that you followed to convert the pages from the Knowledge Base to Markdown? I did an experiment when I built this POC and was able to extract all the Knowledge Base pages in a .zip file (however, the HTML file names included both the title of the page as well as the identifier, such as `Badges (212799365).html`, which could be helpful in telling which file corresponds to which page.

Then I used Pandoc to convert the HTML to Markdown. Note the `--atx-headers` parameter to enforce [ATX-style headers](https://github.github.com/gfm/#atx-headings):

```
$ pandoc -s -t markdown_strict --atx-headers -o test.md "Badges (212799365).html"
$ cat test.md
# Badges

Codacy supports badges you can integrate into your repositories to
show your repository code quality level. 

[...]
```

I very much prefer using ATX-style headers as they're easier to edit manually. Do you mind resubmitting the Markdown files with this style instead?

File: mkdocs.yml (hunk 1 above)
Comment (mrfyda) [reply mkdocs.yml]: I couldn't generate the docs with this line 😅

Comment (pauloribeiro-codacy) [reply mkdocs.yml]: Yes, I had a `theme` folder on my local branch for doing an experiment but since it was empty it was not committed to GitHub.

**Pull request merged**